LayoutTests/fast/css3-text/css3-text-justify/getComputedStyle/getComputedStyle-text-justify.html

```diff
@@ -23,35 +23,35 @@
   {
       debug("Value '" + value + "':");
       e.style.webkitTextJustify = value;
-      testElementStyle("webkitTextJustify", "-webkit-text-justify", "[object CSSPrimitiveValue]", value);
-      testComputedStyle("webkitTextJustify", "-webkit-text-justify", "[object CSSPrimitiveValue]", value);
+      testElementStyle("webkitTextJustify", "text-justify", "[object CSSPrimitiveValue]", value);
+      testComputedStyle("webkitTextJustify", "text-justify", "[object CSSPrimitiveValue]", value);
       debug('');
   }
   
   function invalidValueSettingTest(value, defaultValue)
   {
       debug("Invalid value test - '" + value + "':");
       e.style.webkitTextJustify = value;
-      testElementStyle("webkitTextJustify", "-webkit-text-justify", "[object CSSPrimitiveValue]", defaultValue);
-      testComputedStyle("webkitTextJustify", "-webkit-text-justify", "[object CSSPrimitiveValue]", defaultValue);
+      testElementStyle("webkitTextJustify", "text-justify", "[object CSSPrimitiveValue]", defaultValue);
+      testComputedStyle("webkitTextJustify", "text-justify", "[object CSSPrimitiveValue]", defaultValue);
```

Comment:
```suggestion
      testElementStyle("textJustify", "text-justify", "[object CSSPrimitiveValue]", defaultValue);
      testComputedStyle("textJustify", "text-justify", "[object CSSPrimitiveValue]", defaultValue);
```

This is the reason this test and the other one is failing fwiw.